tools/create_vocabulary.py

```diff
@@ -2,7 +2,6 @@
 # -*- coding: utf-8 -*-
 import argparse
 import sys
-import os
 
 
```

Comment:
maybe rename this script extract_vocabulary or convert_vocabulary ?

Reply:
Good idea!